### PR TITLE
Feature: Added upload pending action

### LINKED ISSUE #84: Register upload in PendingActions plugin
See https://github.com/ckeditor/ckeditor5-core/issues/126.

### PR DESCRIPTION
**Pull request by oskarwrobel**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Added upload pending action. Closes #84.

---

Requires: https://github.com/ckeditor/ckeditor5-core/pull/127.

### CODE REVIEW

File: src/filerepository.js [hunk 1]
```diff
@@ -205,6 +226,25 @@ export default class FileRepository extends Plugin {
...
 	}
+
+	/**
+	 * Registers or deregisters pending action bound with upload progress.
+	 *
+	 * @private
+	 */
+	_updatePendingAction() {
+		const pendingActions = this.editor.plugins.get( PendingActions );
+		const getMessage = value => `Upload in progress ${ parseInt( value ) }%.`;
```
Comment (oskarwrobel) [inline src/filerepository.js]: Translate service should be used.

Comment (pjasiun) [reply src/filerepository.js]: Also, you don't need to define it every time. Only if you plan to add peding action, so I would move it inside `if` below, closer to the place where it is used.

Comment (oskarwrobel) [reply src/filerepository.js]: This is a function.

Comment (oskarwrobel) [reply src/filerepository.js]: Or you mean that this function doesn't have to be defined every time?

Comment (oskarwrobel) [reply src/filerepository.js]: But it shoudl be moved to inside if below. 👍

File: tests/filerepository.js [hunk 2]
```diff
@@ -91,6 +92,69 @@ describe( 'FileRepository', () => {
...
+			expect( pendingActions ).to.have.property( 'isPending', false );
+			expect( Array.from( pendingActions ) ).to.length( 0 );
+
+			fileRepository.createLoader( createNativeFileMock() );
+
+			expect( pendingActions ).to.have.property( 'isPending', true );
+			expect( Array.from( pendingActions, action => action.message ) ).to.have.members( [ 'Upload in progress 0%.' ] );
+		} );
+
+		it( 'should add only one pending action ai a case of multiple load', () => {
```
Comment (oskarwrobel) [inline tests/filerepository.js]: ai

**Pull request merged**